### PR TITLE
Make clear what $ is during the first encounter of this symbol in the…

### PR DESCRIPTION
**Pull request by prominentdetail**
… guide.

Make clear what $ is during the first encounter of this symbol in the guide.

### CODE REVIEW

File: getting_started/step_by_step/signals.rst [hunk 1]
```diff
@@ -103,6 +103,8 @@ the signal is received. Let's make the Sprite blink:
         }
     }
 
+The $ is shorthand for get_node("NodePath").
```
Comment (Calinou) [inline getting_started/step_by_step/signals.rst]: I suggest rewording it to refer directly to the example :slightly_smiling_face: 

```suggestion
``$Sprite`` is a shorthand for ``get_node("Sprite")``.
```

**Comment by mhilbrunner** [comment]
Hey! Thanks for this. This should go against the `master` branch, not the `3.1` branch :)

**Comment by akien-mga** [comment]
This should indeed be made against the master branch, and this shorthand is specific to GDScript so I added a comment directly in the GDScript code block in 940df09.
